samples/azure-quantum/hello-world/HW-ionq-qsharp.ipynb

```diff
@@ -297,11 +298,14 @@
   }
  ],
  "metadata": {
+  "interpreter": {
+   "hash": "81794d4967e6c3204c66dcd87b604927b115b27c00565d3d43f05ba2f3a2cb0d"
+  },
   "kernel_info": {
    "name": "python3"
   },
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.9.5 64-bit",
```

Comment:
All of these small other changes are making me a bit nervous--can we just make the changes to the markdown text itself?

Reply:
This looks like the result of running in a different Python kernel, causing metadata associated with the kernel to be updated — did you run in VS Code notebooks, per chance?

Reply:
Oh yeah that's not intentional, must be due to my environment...will update.

Reply:
@cgranade i did edit in vscode. I suppose that's why.